force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -531,4 +541,25 @@ public inherited sharing class RD2_RecurringDonation {
         return paymentMethod() == RD2_Constants.PAYMENT_PICKLIST_VALUE_CARD ||
                paymentMethod() == RD2_Constants.PAYMENT_PICKLIST_VALUE_ACH;
     }
+
+    /**
+    * @description Return the value of the provided recurring donation field
+    * @param rdField SObjectField to get the value for
+    * @return Object
+    */
+    public Object getValue(SObjectField rdField){
+        return rd.get(rdField);
+    }
+
+    /**
+    * @description Return true if the value of the provided field is different between this
+    * Recurring Donation and the oldRd that was passed in
+    * @param rdField SObjectField to get the value for
+    * @param oldRd RD2_RecurringDonation The Recurring Donation to compare to
+    * @return Boolean
+    */
+    public Boolean fieldChanged(SObjectField rdField, RD2_RecurringDonation oldRd){
```

Comment:
could we add the "is" prefix here to clarify that this is a boolean?

Reply:
Definitely, great catch.